docs: note that bot-generated PRs are ignored by default in Code Reviews (#12572)
Kilo-Org/kilocode: packages/kilo-docs/pages/automate/code-reviews/overview.md

Reviews are posted directly in your platform (GitHub or GitLab) as if coming from a team reviewer.

{% callout type="info" title="Bot-generated PRs are ignored by default" %}

> Kilo does not automatically review pull or merge requests opened by bots, such as Dependabot, Renovate, or other automation accounts. This keeps review credits and notifications focused on human-authored changes. I

Author: ```suggestion
Kilo does not automatically review pull or merge requests opened by bots, such as Dependabot, Renovate, or other automation accounts. This keeps review credits and notifications focused on human-authored changes.
```

> Kilo does not automatically review pull or merge requests opened by bots, such as Dependabot, Renovate, or other automation accounts. This keeps review credits and notifications focused on human-authored changes. If you want bot PRs reviewed, request this on the Code Reviews beta Discord channel.

Author: ```suggestion
Kilo does not automatically review pull or merge requests opened by bots, such as Dependabot, Renovate, or other automation accounts. This keeps review credits and notifications focused on human-authored changes. I
```
